components/Nav/Nav.test.tsx

```diff
@@ -0,0 +1,132 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import Nav from './Nav';
+import Login from '@/app/login/page';
+
+const mockPush = jest.fn();
+const mockUsePathname = jest.fn();
+const mockLogoutAccount = jest.fn();
+
+const mockUseAuthContext = {
+  logoutAccount: mockLogoutAccount,
+  isSignedIn: true,
+};
+
+window.scrollTo = jest.fn();
+
+jest.mock('next/navigation', () => ({
+  useRouter() {
+    return {
+      push: mockPush,
+    };
+  },
+  usePathname() {
+    return mockUsePathname();
+  },
+}));
+
+jest.mock('../../context/AuthContextProvider', () => ({
+  useAuthContext() {
+    return {
+      ...mockUseAuthContext,
+    };
+  },
+}));
+
+Object.defineProperty(window, 'matchMedia', {
+  writable: true,
+  value: jest.fn().mockImplementation((query) => ({
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // deprecated
+    removeListener: jest.fn(), // deprecated
+    addEventListener: jest.fn(),
+    removeEventListener: jest.fn(),
+    dispatchEvent: jest.fn(),
+  })),
+});
+
+describe('Nav', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('it should render the default component state', () => {
+    mockUsePathname.mockImplementation(() => '/weeklyPicks');
+
+    render(<Nav />);
+
+    const navElement = screen.getByTestId('nav');
+    expect(navElement).toBeInTheDocument();
+
+    const drawerTrigger = screen.getByTestId('drawer-trigger');
+    fireEvent.click(drawerTrigger);
+
+    const title = screen.getByText(/gridiron survivor/i);
```

Comment:
Why are we grabbing this element by the text, not `data-testid` like navElement and drawerTrigger?